source/manage/server-admin-gui.rst

```diff
@@ -28,6 +28,43 @@ as well as configure `telemetry <https://www.percona.com/doc/percona-monitoring-
 
 Don't forget to click the *Apply changes* button to make changed options work.
 
+.. _server-admin-gui-telemetry:
+
+Telemetry
+================================================================================
+
+The *Telemetry* switch enables gathering and sending back to Percona basic
+**anonymous** data, which helps us to determine where to focus the development
+and what is the uptake of the various versions of PMM. Particularly, this
+information helps to determine if we need to release patches to legacy versions
+that may not technically be supported all the way to how the frequency of the
+release impacts adoption.
```

Comment:
Specifically, gathering this information helps determine if we need to release patches to legacy versions beyond support, determining when supporting a particular version is no longer necessary, and even understanding how the frequency of release encourages or deters adoption.